### ISSUE TITLE
Expressions that involve nested func calls get wrapped more than they should

### ISSUE THREAD

**Issue by Rassi** [uncrustify/uncrustify#1392]
```
A(B(C(
    D(a |
        b | c))));
```

Results in an indentation of "D" for each nested func.

```
A(B(C(
            D(a |
                b | c))));
```

What we want instead is what is in the original - each newline should cause a single indent, regardless of nesting.

Expected versus output diff:
---
```diff
diff --git "a/C:\\dev\\uncrustify\\tests\\output\\staging\\60036-UNI-2680.cpp" "b/C:\\dev\\uncrustify\\tests\\results\\staging\\60036-UNI-2680.cpp"
index b6aa5bd..1dfba96 100644
--- "a/C:\\dev\\uncrustify\\tests\\output\\staging\\60036-UNI-2680.cpp"
+++ "b/C:\\dev\\uncrustify\\tests\\results\\staging\\60036-UNI-2680.cpp"
@@ -1,3 +1,3 @@
 A(B(C(
-    D(a |
-        b | c))));
+            D(a |
+                b | c))));
```

Debug support files:
---
[60036-UNI-2680.cpp](https://gist.githubusercontent.com/Rassi/fe513482b94d7bd426685c82cd71ef2c/raw/b6aa5bd73eada819d5508002427adeabdb5c2f3c/60036-UNI-2680.cpp)
[60036-UNI-2680.cpp.log](https://gist.githubusercontent.com/Rassi/fe513482b94d7bd426685c82cd71ef2c/raw/0c3c5e34d6183638a25bbc1a4a60b119c6c20b09/60036-UNI-2680.cpp.log)
[60036-UNI-2680.cpp.unc](https://gist.githubusercontent.com/Rassi/fe513482b94d7bd426685c82cd71ef2c/raw/5bf1e51aab1f79c4bbc9c291c6b7e020e912b2f8/60036-UNI-2680.cpp.unc)
[Uncrustify.Cpp.cfg](https://gist.githubusercontent.com/Rassi/fe513482b94d7bd426685c82cd71ef2c/raw/23d62d9dba5caf352fdc1793792e98eed4b81929/Uncrustify.Cpp.cfg)
[UNI-2680.cpp](https://gist.githubusercontent.com/Rassi/fe513482b94d7bd426685c82cd71ef2c/raw/b6aa5bd73eada819d5508002427adeabdb5c2f3c/UNI-2680.cpp)

Disclaimer:
---
Note: This is a generated issue from our internal issue tracker.
Expect a dry description which could be insufficient to properly understand the issue.
On the other hand, the support files should be sufficient to easily reproduce and debug the issue,
but please comment if it's still unclear and I will get back with clarifications.
See main discussion thread here #1223.

**Comment by mihaipopescu** [comment]
This is #1252 but with correct input/output.

**Comment by satishp87** [comment]
In unity fork this issue is addressed with below PR
https://github.com/Unity-Technologies/uncrustify/pull/39

**Comment by gmaurel** [comment]
Fixed at 337b538da318f3bec5a4ca5be64e4ad2af784552, Dec 28 16:02:40 2017
